pkgs/by-name/au/autoconf-archive/package.nix

```diff
@@ -9,6 +9,15 @@ stdenv.mkDerivation rec {
     hash = "sha256-e81dABkW86UO10NvT3AOPSsbrePtgDIZxZLWJQKlc2M=";
   };
 
+  patches = [
+    # fix https://bugs.gentoo.org/941845
+    (fetchurl {
```

Comment:
what is the reason for using `fetchurl` here? `fetchpatch`/`fetchpatch2` should be preferred when fetching patches generated by git frontends

Reply:
I believe there's some bootstrap context where this package is evaluated, but `fetchpatch` is broken.  I think because the bootstrap `fetchurl` is incompatible with it?  It's easy to repro by evaluating this package for linux with the change.

Reply:
If there would be a circular dependency with a fetcher, we generally include patches directly in Nixpkgs. Using `fetchurl` for patches means that the patch might become unreproducible if the upstream service changes

Reply:
I've updated this to include the patches in nixpkgs.  There were two other similar patches that were merged together, so I grabbed them all.

I think that's a good idea given these are small and it's a critical package.